src/profile-logic/gecko-profile-versioning.js

```diff
@@ -980,14 +980,15 @@ const _upgraders = {
                 newStartTime = time;
                 newEndTime = null;
                 phase = INTERVAL_START;
-              } else {
+              } else if (interval === 'end') {
                 newStartTime = null;
                 newEndTime = time;
                 phase = INTERVAL_END;
+              } else {
+                // The interval property could also be inexistant. In that case we
+                // decide this is an instance marker.
               }
             } else if (
-              // This could be considered an instant marker, since the startTime and
-              // endTime are the same.
```

Comment:
This comment looked outdated

Reply:
Is it? I think this comment still applies to that if check, it explains why we have `startTime !== endTime`.